docker-compose.yml

```diff
@@ -123,6 +123,7 @@ services:
             - JIBRI_BREWERY_MUC
             - JIBRI_PENDING_TIMEOUT
             - TZ
+            - JICOFO_RESERVATION_REST_BASE_URL
```

Comment:
small nit: do you mind moving this up, together with the other Jicofo variables? Thank you!

Reply:
Sure and done! :)